debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -990,7 +990,7 @@ protected void handleDataEvent(LogMinerEventRow row) throws SQLException, Interr
             return;
         }
 
-        LOGGER.trace("DML: {}", row);
+        LOGGER.debug("DML: {}", row);
```

Comment:
@Naros Is not this exactly the type of messages that makes the privacy concerns?

Reply:
Feel free to merge yourself when this and the similar cases are addressed

Reply:
No, I modified the `LogMinerEventRow` `toString()` method to explicitly exclude the `redoSql` unless the log level was at least `TRACE`; otherwise, it replaces the SQL with `<omitted>` but keeps the remainder of the output identical so we get the other metadata and attributes about the row which isn't sensitive.